tests/unit/manage/test_views.py

```diff
@@ -379,8 +379,8 @@ def test_change_primary_email(self, monkeypatch, db_request):
         user = UserFactory()
         old_primary = EmailFactory(primary=True, user=user)
         new_primary = EmailFactory(primary=False, verified=True, user=user)
-
```

Comment:
Also, this is a very small issue, but it seems like your PR removes some empty lines like this one here -- could you make sure they remain?